scripts/build

```diff
@@ -255,9 +270,17 @@ rm -rf {,share}/{info,man,doc} || exit 60
 # Strip libraries and binaries
 #
 
-strip --strip-debug    {,$TARGET/}lib/* > /dev/null
-strip --strip-unneeded {,$TARGET/}bin/* > /dev/null
+if [[ $(uname -s) = "Linux" ]]; then
+  strip --strip-debug    {,$TARGET/}lib/* > /dev/null
+  strip --strip-unneeded {,$TARGET/}bin/* > /dev/null
+  strip --strip-unneeded libexec/gcc/$TARGET/*/*{,/*} > /dev/null
+fi
 
-strip --strip-unneeded libexec/gcc/$TARGET/*/*{,/*} > /dev/null
+# Functional equivalents for OS X
+if [[ $(uname -s) = "Darwin" ]]; then
+  strip -u -r -S {,$TARGET/}lib/* > /dev/null
+  strip -u -r -x {,$TARGET/}bin/* > /dev/null
+  strip -u -r -x libexec/gcc/$TARGET/*/*{,/*} > /dev/null
+fi
```

Comment:
Maybe these could be checked on the top of the file and set to `$STRIP_DEBUG` and `$STRIP_UNNEEDED` variables similar to the `$RMDIR` I tell you before...